### DISCUSSION TITLE
[REQ] Feature, "Show expected filename" for import / renames

### DISCUSSION THREAD

**Discussion by jaxjexjox** [pymedusa/Medusa#9642] [Ideas & Feature requests]
Hi,

Sometimes I have NO IDEA what the right filename should be, to import a TV show.    
I do know about the scene exceptiona naming, but it would be nice to just be able to provide files with an approximate name close to the right one.

Totally random example

Dexter (2006)  S01E01
vs
Dexter S01E01

or perhaps

Attack on Titan (Shingeki no Kyojin)  S01E01
vs
Attack on Tital S01E01

etc etc,  I hope this makes sense.

Prefer in the section where it lists Quality, Location, Ignored Words, Size (etc0  - but another field
"Expected filename :   TV Show (Year) " etc

**Comment by jaxjexjox** [comment]
Hello, has anyone by any chance been able to see this? It would be appreciated.

Thanks for the hard work.

**Reply by p0psicles** [reply]
Hi, no we don't have anything like that.
We do have a new feature since late. Where you can test a release name using guessit. Which is the tool we use to parse release names into title, season, episode, etc.

I could extend it with a feature where you can test the title against a show in medusa?

**Comment by p0psicles** [comment]
This is what i've implemented now.
![image](https://user-images.githubusercontent.com/1331394/154650411-fa588118-7fcc-4737-b359-122af9da3f85.png)

**Comment by jaxjexjox** [comment]
The ideal solution though, would be for the program to tell me EXACTLY what it's going to RENAME a file TO (destination)

I am having a misery of a time right now, importing a CBS show called Ghosts (2021)

It's either "Ghosts.2021" or "Ghosts (US)" or something but Medusa is being quite stubborn importing it.

If I could just force the package to tell me "I will end up, calling it EXACTLY this...." then that would make life easier. (I appreciate your help!)
